java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md

```diff
@@ -0,0 +1,82 @@
+## 1. Kafka To BigQuery
+
+General Execution:
+
+```
+export GCP_PROJECT=<gcp-project-id>
+export REGION=<gcp-project-region>
+export GCS_STAGING_LOCATION=<gcs-bucket-staging-folder-path>
+export SUBNET=<gcp-project-dataproc-clusters-subnet>
+
+bin/start.sh \
+-- \
+--template KAFKATOBQ \
+--templateProperty project.id=$GCP_PROJECT \
+--templateProperty kafka.bq.checkpoint.location=<gcs-bucket-location-maintains-checkpoint> \
+--templateProperty kafka.bq.bootstrap.servers=<kafka-broker-list> \
+--templateProperty kafka.bq.topic=<kafka-topic-names> \
+--templateProperty kafka.bq.starting.offset=<starting-offset-value> \
+--templateProperty kafka.bq.dataset=<output-bigquery-dataset> \
+--templateProperty kafka.bq.table=<output-bigquery-table> \
+--templateProperty kafka.bq.temp.gcs.bucket=<gcs-bucket-name> \
+--templateProperty kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
+```
+
+### Configurable Parameters
+Following properties are avaialble in commandline or [template.properties](../../../../../../../resources/template.properties) file:
+
+```
+# Kafka to BigQuery
+
+# Kafka servers
+kafka.bq.bootstrap.servers=<kafka broker list>
+
+# Kafka topics
+kafka.bq.topic=<kafka topic names>
+
+# BigQuery output dataset
+kafka.bq.dataset=<output bigquery dataset>
+
+# BigQuery output table
+kafka.bq.table=<output bigquery table>
+
+# GCS bucket name, for storing temporary files
+kafka.bq.temp.gcs.bucket=<gcs bucket name>
+
+# GCS location for maintaining checkpoint
+kafka.bq.checkpoint.location=<gcs bucket location maintains checkpoint>
+
+# Offset to start reading from. Accepted values: "earliest", "latest" (streaming only), or json string """ {"topicA":{"0":23,"1":-1},"topicB":{"0":-2}} """
```

Comment:
What is meant by "streaming only".
Can this be done somehow in batch mode too ?

Reply:
@shashank-google  As discussed previously the template is designed to support both batch loads as well as streaming loads.
The spark property when set to earliest, the template batch loads the data, in other terms instructs the spark query to read from the earliest offsets.
Whereas when set to latest, the template serves to the streaming loads, basically instructing the query to start read from the latest offset.

Modified the README.md to be more descriptive on these new spark properties